app/knight_class.py

```diff
@@ -0,0 +1,34 @@
+class Knight:
+    """Class creates the knight instance and necessary methods"""
+
+    def __init__(self, name: str, knights: dict) -> None:
+        """Method constructs knight instance"""
+        self.name = name
+        self.real_name = knights[name]["name"]
```

Comment:
I don’t think you need those `name` and `real_name` different attributes. Please provide only `name` all over the project

Reply:
OK